README.md

```diff
@@ -325,6 +326,7 @@ Please read [CONTRIBUTING](./CONTRIBUTING.md) if you wish to add software.
 *LDAP management*
 
 * [Apache Directory Studio](https://directory.apache.org/studio/) - The Eclipse-based LDAP browser and directory client
+* [LDAP Admin](http://www.ldapadmin.org/) - A Windows GUI for LDAP administration.
```

Comment:
that does not belong here.

Reply:
ah so sorry, i wanted to submit two seperate pull requests for both commits, can i remove one (i don't have much experience with github).

Reply:
Clone the repo, revert the change, ammend it to the commit, force push